src/Nethermind/Nethermind.Network.Discovery/Discv5/DiscoveryV5App.cs

```diff
@@ -57,50 +60,44 @@ public DiscoveryV5App(SameKeyGenerator privateKeyProvider, IApiWithNetwork api,
 
         IServiceCollection services = new ServiceCollection()
            .AddSingleton<ILoggerFactory, NullLoggerFactory>()
-           .AddSingleton(sessionOptions.Verifier)
-           .AddSingleton(sessionOptions.Signer);
+           .AddSingleton(_sessionOptions.Verifier)
+           .AddSingleton(_sessionOptions.Signer);
 
         EnrFactory enrFactory = new(new EnrEntryRegistry());
 
         Lantern.Discv5.Enr.Enr[] bootstrapEnrs = [
             .. bootstrapNodes.Where(e => e.StartsWith("enode:"))
                 .Select(e => new Enode(e))
-                .Select(e => new EnrBuilder()
-                    .WithIdentityScheme(sessionOptions.Verifier, sessionOptions.Signer)
-                    .WithEntry(EnrEntryKey.Id, new EntryId("v4"))
```

Comment:
@flcl42, why have you decided to use `"v4"` here?
- If it's a node id - shouldn't it be unique for each one?
- If it's a protocol-id - I think it should be `"discv5"`, according to the [doc](https://github.com/ethereum/devp2p/blob/master/discv5/discv5-wire.md).

Reply:
It's identity scheme for enr, not for discv5 packets. And for discv5 it's [v4](https://github.com/ethereum/devp2p/blob/master/discv5/discv5-wire.md#handshake-message-packet-flag--2), which also mentioned [here](https://github.com/ethereum/devp2p/blob/master/discv5/discv5-wire.md#handshake-message-packet-flag--2) (=enr has v4 scheme)